apps/web/src/components/Table/ApiManagedTable/useFilterOptions.tsx

```diff
@@ -33,6 +34,7 @@ import {
   CandidateExpiryFilter,
   CandidateSuspendedFilter,
 } from "~/api/generated";
+import { PoolStream } from "@gc-digital-talent/graphql";
```

Comment:
This should probably be imported with all the other models above.  Either move them to this import or move this model to that import.